README.md

```diff
@@ -126,16 +126,20 @@ For more examples, see the [examples](examples) directory. For more information
 
 ### Create a model
 
-`ollama create` is used to create a model from a Modelfile.
+Create a model from a Modelfile
```

Comment:
We can remove this section for the `Customize a Prompt` section below

Reply:
removed in upcoming commit